File: ruby/arrays/operations_on_elements/popping_elements.rb
#!/usr/bin/ruby2.7

array = [2, 4, 6, 8, 10]

puts("Initial state:\n\n")
puts(array.inspect, "\n")

array.pop()

puts("After pop by array.pop():\n\n")
puts(array.inspect, "\n")

array.pop(0)

puts("After pop by array.pop(0):\n\n")
puts(array.inspect, "\n")

array.pop(1)

puts("After pop by array.pop(1):\n\n")
puts(array.inspect, "\n")

array.pop(2)

puts("After pop by array.pop(2):\n\n")
puts(array.inspect, "\n")

File: php/arrays/operations_on_elements/popping_elements.php
<?php
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

$array = [2, 4, 6];

print("Initial state:\n\n");
print_r($array); print("\n");

// array_pop

array_pop($array);

print("After pop by array_pop(\$array):\n\n");
print_r($array); print("\n");
